### ISSUE TITLE
Kick from server with Mundane Potion

### ISSUE THREAD

**Issue by xLivan** [Vazkii/ReCubed#17]
This happens when throwing Mundane Potion.
```
java.lang.NullPointerException
	at vazkii.recubed.common.core.handler.GeneralEventHandler.onPlayerInteract(GeneralEventHandler.java:246) ~[GeneralEventHandler.class:?]
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_505_GeneralEventHandler_onPlayerInteract_PlayerInteractEvent.invoke(.dynamic) ~[?:?]
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54) ~[ASMEventHandler.class:1.7.10-1614.188]
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140) ~[EventBus.class:1.7.10-1614.188]
	at net.minecraftforge.event.ForgeEventFactory.onPlayerInteract(ForgeEventFactory.java:100) ~[ForgeEventFactory.class:1.7.10-1614.188]
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:817) ~[nh.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:245) ~[ej.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:173) [nc.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:991) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:431) [lt.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:809) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:669) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_51]
```

**Comment by xLivan** [comment]
#18

**Comment by Vazkii** [comment]
Fixed in 1.9
